### DISCUSSION TITLE
How to use ganache-cli over local RPC network brownie

### DISCUSSION THREAD

**Discussion by evancatania13** [smartcontractkit/full-blockchain-solidity-course-py#872] [Q&A] [answered]
Hi all, 
At the brownie section of the course (testing step) and cannot get rid of the warning 

"BrownieEnvironmentWarning: Development network has a block height of 15"

I think this has something todo with the fact that I cannot connect to the ganache-cli for some reason, instead brownie is using a local RPC client listening at the same port.

While this isn't breaking anything it'd be great if anyone knew how to either close the RPC connection to allow for the ganache-cli to be used or how to change networks.

*Adding*

I noticed in Patrick's examples there is a line after he runs his tests "Terminating Local RPC Client", I am also not getting that. 

Thanks!

**Comment by cromewar** [comment, marked as answer]
Hello @catanman-eng it depends a lot on your installation, I suggest the following.

1. uninstall and install again node using nvm (install a lts version: 16)
2. reinstall ganache within 
3. run ganache cli and see if it works properly now.
4. if not please share the error here

**Reply by evancatania13** [reply]
Seems to have just reset the block height 
Still get the error but it isn't breaking anything just a visual thing. 
I'll mark as answer for now
